trajpy/gui_animals.py

```diff
@@ -0,0 +1,174 @@
+from tkinter import *
+from tkinter import filedialog
+import animals
+import live_tracking
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.collections import LineCollection
+from matplotlib.colors import LinearSegmentedColormap
+from matplotlib import cm
+
+root = Tk()
+root.title('TrajPy animals GUI')
+root.geometry('470x350')
+root.resizable(False, False)
+
+
+
+
+results = []
+features = []
+
+def track_function():
+    global file_variables
```

Comment:
- Documentation needed.  
- Why are you defining a global variable inside a function? This looks confusing. Please check the guidelines.

Reply:
The documentation is ready, but it's quite extensive. I've written it inside the file itself just as we do in trajpy.py to describe a function.

The global variable is there cause that's the only way I could make it work, but I'll try to make a Class out of the code so that I don't need global variables.

Reply:
The documentation is ready, but it's quite extensive. I've written it inside the file itself just as we do in trajpy.py to describe a function.

The global variable is there cause that's the only way I could make it work, but I'll try to make a Class out of the code so that I don't need global variables.